package.json

```diff
@@ -11,7 +11,8 @@
     "test": "nps test",
     "test:ts": "(tsc -p ./tsconfig.json && rimraf test-ts) || rimraf test-ts",
     "commitmsg": "opt --in commit-msg --exec \"validate-commit-msg\"",
-    "precommit": "lint-staged && opt --in pre-commit --exec \"npm start validate\""
+    "precommit": "lint-staged && opt --in pre-commit --exec \"npm start validate\"",
+    "doctoc": "doctoc --maxlevel 3 --title '**Table of Contents**' README.md"
```

Comment:
Sorry, basically, you can do the same thing as we're doing with the `lint` script [here](https://github.com/paypal/glamorous/blob/b7424288431878bace8d9bb3b7d641e21a61b78f/package-scripts.js#L70-L73). So you can add:

```javascript
    doctoc: {
      description: 'generates a Table of Contents in the README',
      script: 'doctoc --maxlevel 3 --title '**Table of Contents**' README.md',
    },
```

Then you can run that script with: `npm start doctoc` :+1: